backend/data/processors/manualFixes/ECONFixes.py

```diff
@@ -0,0 +1,192 @@
+COURSES = dataHelpers.read_data("data/finalData/coursesProcessed.json")
+
+def fix_conditions():
+    """ Functions to apply manual fixes """
+
+    # TODO: call your functions here
+    CONDITIONS["ECON1101"][PROCESSED] = ECON_1101("ECON1101") # NOT SURE HOW THIS WORKS (FOR THE EXCEPT PART)
+    CONDITIONS["ECON1203"][PROCESSED] = ECON_1203("ECON1203") # NOT SURE HOW THIS WORKS (FOR THE EXCEPT PART)
+    CONDITIONS["ECON2206"][PROCESSED] = ECON_2206()
+    CONDITIONS["ECON2403"][PROCESSED] = ECON_2403()
+    CONDITIONS["ECON3000"][PROCESSED] = ECON_3000()
+    CONDITIONS["ECON3208"][PROCESSED] = ECON_3208()
+    CONDITIONS["ECON4100"][PROCESSED] = ECON_4100_3_6_4202_5_8_4301_2_7_9_11_50()
+    CONDITIONS["ECON4103"][PROCESSED] = ECON_4100_3_6_4202_5_8_4301_2_7_9_11_50()
+    CONDITIONS["ECON4106"][PROCESSED] = ECON_4100_3_6_4202_5_8_4301_2_7_9_11_50()
+    CONDITIONS["ECON4150"][PROCESSED] = ECON_4150_2()
+    CONDITIONS["ECON4151"][PROCESSED] = ECON_4150_2()
+    CONDITIONS["ECON4152"][PROCESSED] = ECON_4150_2()
+    CONDITIONS["ECON4160"][PROCESSED] = ECON_4160_2()
+    CONDITIONS["ECON4161"][PROCESSED] = ECON_4160_2()
+    CONDITIONS["ECON4162"][PROCESSED] = ECON_4160_2()
+    CONDITIONS["ECON4170"][PROCESSED] = ECON_4170_2()
+    CONDITIONS["ECON4171"][PROCESSED] = ECON_4170_2()
+    CONDITIONS["ECON4172"][PROCESSED] = ECON_4170_2()
+    CONDITIONS["ECON4201"][PROCESSED] = ECON_4201() # NOT SURE ABOUT ASSUMED KNOWLEDGE
+    CONDITIONS["ECON4202"][PROCESSED] = ECON_4100_3_6_4202_5_8_4301_2_7_9_11_50()
+    CONDITIONS["ECON4205"][PROCESSED] = ECON_4100_3_6_4202_5_8_4301_2_7_9_11_50()
+    CONDITIONS["ECON4208"][PROCESSED] = ECON_4100_3_6_4202_5_8_4301_2_7_9_11_50()
+    CONDITIONS["ECON4301"][PROCESSED] = ECON_4100_3_6_4202_5_8_4301_2_7_9_11_50()
+    CONDITIONS["ECON4302"][PROCESSED] = ECON_4100_3_6_4202_5_8_4301_2_7_9_11_50()
+    CONDITIONS["ECON4303"][PROCESSED] = ECON_4303()
+    CONDITIONS["ECON4307"][PROCESSED] = ECON_4100_3_6_4202_5_8_4301_2_7_9_11_50()
+    CONDITIONS["ECON4309"][PROCESSED] = ECON_4100_3_6_4202_5_8_4301_2_7_9_11_50()
+    CONDITIONS["ECON4310"][PROCESSED] = ECON_4100_3_6_4202_5_8_4301_2_7_9_11_50()
+    CONDITIONS["ECON4311"][PROCESSED] = ECON_4100_3_6_4202_5_8_4301_2_7_9_11_50()
+    CONDITIONS["ECON4350"][PROCESSED] = ECON_4100_3_6_4202_5_8_4301_2_7_9_11_50()
+
+    # Updates the files with the modified dictionaries
+    dataHelpers.write_data(
+        CONDITIONS, "data/finalData/conditionsProcessed.json")
+    dataHelpers.write_data(COURSES, "data/finalData/coursesProcessed.json")
+
+# TODO: implement your functions here
+# NOT SURE HOW THIS WORKS (FOR THE EXCEPT PART)
+def ECON_1101(code):
+    """
+    "original": "Excluded: Students in BCom single or double-degree programs, except for 3155 Actl/Comm and 3521 Comm/Econ, are not allowed to enrol in this course. <br/><br/>"
+    
+    "processed": ""
+    """
+
+    COURSES[code]["exclusions"]["COMM#"] = 1
```

Comment:
This condition is so doomed, I think one possibility is to not add the "COMM#" to the exclusion and instead just return that as a warning: "Students in BCom single or double-degree programs, except for 3155 and 3521 cannot enrol"

This way we don't restrict eligible students but still warn ineligible students